crowdin.yaml

```diff
@@ -1,19 +1,3 @@
-project_identifier: django-girls-tutorial
-api_key: 2be4b1707d0745bc96e9a5c9ffb4ca9b
-base_path: /Users/olasitarska/Desktop/Events/Django Girls/tutorial
-
 files:
-  -
-    source: "/en/**/*.md"
-    translation: "/%two_letters_code%/%original_path%/%original_file_name%"
```

Comment:
Assuming that we will leave this line here translation of `/en/beginning/start.md` will be saved in `/pl/en/beginning/start.md`.

My change will save translation in `/pl/beginning/start.md` (which is compatible with structure of Github repo).